File: java-main/java-main/IO/Nestedfor5/p3.java
/*write a program to print the following pattern
5 4 3 2 1
8 6 4 2
9 6 3
8 4
5
*/

import java.util.Scanner;
class Demo{
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the number of rows: ");
        int row = input.nextInt();

        int num = 5;
        for(int i = 1; i <= row; i++) {
            for(int j = row; j >= i; j--) {
                System.out.print(num + " ");
                num--;
            }
            System.out.println();
        }

        num = 8;
        for(int i = 1; i <= row; i++) {
            for(int j = row - i; j >= 1; j--) {
                System.out.print(num + " ");
                num -= 2;
            }
            System.out.println();
            num += (i * 2) + 1;
        }
    }
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/session3/p3.c
#include<stdio.h>
void main(){

	int rows;
	int num=1;

	printf("Enter no of rows:\n");
	scanf("%d",&rows);

	for(int i=1;i<=rows;i++){

		int num=i;

		for(int j=1;j<=i;j++){
			printf("%d ",num);
		}
		num++;
		printf("\n");
	}
}


